Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -54,6 +54,14 @@ def __line__():
 SAFE_SLEEP_START_TIME = datetime.now()
 
 
+if 'context' in demisto.callingContext \
+        and 'ExecutedCommands' in demisto.callingContext['context'] \
+        and 'name' in demisto.callingContext['context']['ExecutedCommands'][0]:
+    context_executed_commands_name = demisto.callingContext['context']['ExecutedCommands'][0]['name']
+    with open('demisto_info.txt', 'w') as file_demisto_info:
```

Comment:
wrap this code in try and except

Reply:
in case the write can fail for some reason

Reply:
[Done](https://github.com/demisto/content/pull/34445/commits/61150ec2085d931fe5b82c47fd7ab0eaf8d4aa9a).
I wrapped the whole thing, in case the structure of `demisto` is changed.